README.md

```diff
@@ -178,3 +178,28 @@ differences.
 
 If you want to update the reference screenshot for a given example, you can grab
 those from the build artifacts as well and commit them to your branch.
+
+### Testing Locally
+
+Testing locally is possible, however pixel perfect results will differ from
+those on the CIs due to discrepencies in hardware, and driver (we use llvmpipe)
+versions.
+
+On linux, it is possible to force to force the usage of LLVMPIPE in the test suite
+and compare the generated results of screenshots. Beware, the results on your machine
+may differ to those on the CI. We always include the CI screenshots in the test suite
+to improve the repeatability of the tests.
+
+If you have access to a linux machine with llvmpipe installed, you may run the
+example pixel comparison testing by setting the WGPUPY_WGPU_ADAPTER_NAME
+environment variable appropriately. For example
+
+
+```
+WGPUPY_WGPU_ADAPTER_NAME=llvmpipe pytest -v examples/
+```
+
+The `WGPUPY_WGPU_ADAPTER_NAME` variable is modeled after the
+https://github.com/gfx-rs/wgpu?tab=readme-ov-file#environment-variables
+and should only be used for testing the wgpupy library itself.
+It is not part of the supported wgpy-py interface.
```

Comment:
Two instances of wgpu-py misspelled in the README would be a bad look. ;) can you correct them?

Reply:
eek soryr.